src/en/wuxiaworldsite/WuxiaWorldSite.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const WuxiaWorldScraper = require("./WuxiaWorldSiteScraper");
+
+const router = express.Router();
+
+router.get("/novels/", WuxiaWorldSiteScraper.novelsScraper);
+
+router.get("/novel/:novelID/:novelUrl", WuxiaWorldSiteScraper.novelScraper);
```

Comment:
Most sources follow the same endpoints for novel and chapter which are `/novel/:novelUrl` and `/novel/:novelUrl/:chapterUrl`  and because of `novelID`  it would not work.  

I checked the website and if we remove the `novelID` from the url the same novel or chapter page still loads so if we just remove the `/:novelID` it would work.

Reply:
I should've noticed that. Thanks for pointing it out.

Reply:
Fixed it. :D